src/Pod.ts

```diff
@@ -695,68 +598,108 @@ export default class Pod {
   };
 
   /**
-   * Mints member to this pod.
+   * Mints member to this pod, or returns an unsigned transaction.
+   * @param signer - If a signer is provided, then the tx will execute. Otherwise, an unsigned transaction will be returned.
    * @throws if signer is not admin
    */
   mintMember = async (
     newMember: string,
-    signer: ethers.Signer,
-  ): Promise<ethers.providers.TransactionResponse> => {
+    signer?: ethers.Signer,
+  ): Promise<ethers.providers.TransactionResponse | { to: string; data: string }> => {
     checkAddress(newMember);
-    const signerAddress = await signer.getAddress();
-    if (!this.isAdmin(signerAddress)) throw new Error('Signer was not admin');
+    if (signer) {
+      const signerAddress = await signer.getAddress();
+      if (!this.isAdmin(signerAddress)) throw new Error('Signer was not admin');
+    }
     try {
-      return getContract('MemberToken', signer).mint(newMember, this.id, ethers.constants.HashZero);
+      const MemberToken = getContract('MemberToken', signer);
+      if (signer) return MemberToken.mint(newMember, this.id, ethers.constants.HashZero);
+      return (await MemberToken.populateTransaction.mint(
+        newMember,
+        this.id,
+        ethers.constants.HashZero,
+      )) as { to: string; data: string };
     } catch (err) {
       return handleEthersError(err);
     }
   };
 
   /**
-   * Burns member from this pod.
+   * Burns member from this pod, or returns an unsigned transaction
+   * @param signer - If a signer is provided, then the tx will execute. Otherwise, an unsigned transaction will be returned.
    * @throws If signer is not admin
    */
   burnMember = async (
     memberToBurn: string,
-    signer: ethers.Signer,
-  ): Promise<ethers.providers.TransactionResponse> => {
+    signer?: ethers.Signer,
+  ): Promise<ethers.providers.TransactionResponse | { to: string; data: string }> => {
     checkAddress(memberToBurn);
-    const signerAddress = await signer.getAddress();
-    if (!this.isAdmin(signerAddress)) throw new Error('Signer was not admin');
+    if (signer) {
+      const signerAddress = await signer.getAddress();
+      if (!this.isAdmin(signerAddress)) throw new Error('Signer was not admin');
+    }
     try {
-      return getContract('MemberToken', signer).burn(memberToBurn, this.id);
+      const MemberToken = getContract('MemberToken', signer);
```

Comment:
what happens if we don't pass a signer?

Reply:
not sure of broader implications (i.e., if you can make read calls against the blockchain), but I checked to make sure it works for populating transactions, because we still have access to the Interface